docs/integrations/saas-cloud/docusign.md

```diff
@@ -159,19 +159,13 @@ _sourceCategory="docusign_src"
 | transpose row _timeslice column severity as low, medium, high
 ```
 
-## Installing the DocuSign App
+## Installing the DocuSign app
 
-Before you begin, collect logs from DocuSign API and ingest them into Sumo Logic. Refer to the [DocuSign API Cloud-to-Cloud Integration](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source/) to create the source and use the same source category while installing the app.
+{@import ../../reuse/apps/app-install-v2.md}
 
-{@import ../../reuse/apps/app-install.md}
+## Viewing DocuSign dashboards
 
-## Viewing the DocuSign Dashboards
-
-All dashboards have a set of filters that you can apply to the entire dashboard, as shown in the following example. Click the funnel icon in the top dashboard menu bar to display a scrollable list of filters that are applied across the entire dashboard.
-
-You can use filters to drill down and examine the data on a granular level. Filters include client country, client device type, client IP, client request host, client request URI, client request user agent, edge response status, origin IP, and origin response status.
-
-Each panel has a set of filters that are applied to the results for that panel only, as shown in the following example. Click the funnel icon in the top panel menu bar to display a list of panel-specific filters.
+{@import ../../reuse/apps/view-dashboards.md}
 
 ### DocuSign - Overview
```

Comment:
Can you just remove the app name from all the dashboard headings.

Reply:
It is the name given to respective dashboards.
And if we remove or modify any name in Doc it may create confusion.

Reply:
Updated doc